src/app/concern/file-uploader/file-uploader.component.ts

```diff
@@ -60,33 +60,13 @@ export class FileUploaderComponent implements OnInit, OnDestroy {
 
   ngOnInit() {
     this.setupForm();
-    this.setConcernId();
   }
 
   public setupForm(): void {
     this.form = this.formBuilder.group({ fileUploader: null });
   }
 
-  /**
-   * subscribe to each concern change and upload files automatically if files are in progress
-   */
-  public setConcernId(): void {
-    this.subsciptions.push(
-      this.selectedConcern$.subscribe((selectedConcern: IConcernSummary) => {
-        this.concernId = selectedConcern.concernId;
-        const filesInProgress = this.store.selectSnapshot(
-          ConcernState.filesInUploadProgress
-        );
-        if (this.concernId && filesInProgress?.length > 0) {
-          const files = filesInProgress.map(
-            (val: IFileUploadProgress) => val.file
-          );
-          this.upload(files);
-        }
-      })
-    );
-  }
-
+  // TODO check if needed why?
```

Comment:
when there's a concern ID the file upload progress will resume here.

Reply:
no worries feel free to update